### DISCUSSION TITLE
Can SSCHA calculate high temperature（20000K）anharmonic free energy？

### DISCUSSION THREAD

**Discussion by SunGroup-nju** [SSCHAcode/python-sscha#128] [Q&A] [answered]
I have a high density and high temperature project，about 100g/cm3 and 20000K，can I use SSCHA to get a accurate result？

**Comment by mesonepigreco** [comment, marked as answer]
Hi, 
It depends on the material.
SSCHA is not very good at describing liquid, so if your material is melted at that temperature, the SSCHA may not be the best tool (and you should use molecular dynamics instead).
However, if you also have high pressures, it is still possible that the material is solid (even if, at those temperatures, it probably is not). 
In that case, the SSCHA is the best tool!
